curvefs/proto/mds.proto

```diff
@@ -155,12 +155,13 @@ message DeleteFsResponse {
 
 message AllocateS3ChunkRequest {
     required uint32 fsId = 1;
+    required uint64 chunkIdNum = 2;
```

Comment:
The required field will lead to incompatible with previous version. A feasible way is to make this field optional, and if client-side sets this field then the server-side allocates more chunk ids, and if not, server-side only allocates one chunk id.
 
And `uint32` is already enough for this field.

Reply:
fixed